kernel/common/get_slice.rs

```diff
@@ -1,8 +1,6 @@
-use core::slice;
-use core::str;
 use collections::range::RangeArgument;
 use core::ops::Range;
-use core::cmp::{max, min};
+use core::cmp::min;
```

Comment:
Erm... That was actually not intented. Interestingly, the diff it exactly correct, and do only introduce changes, I had intented to PR, indicating that `master` and `orbital` is in sync...